Statut.md

```diff
@@ -0,0 +1,169 @@
+	d) dochodów z darów, ze zbiórek i imprez publicznych,
+	e) aukcji internetowych,
+	f) dochodów z majątku nieruchomego i ruchomego,
+	h) odsetek i depozytów bankowych,
+	i) z działalności odpłatnej pożytku publicznego.
+	j) * ... działalności gospodarczej ??? *
+3. Dochody pochodzące z subwencji, darowizn, spadków i zapisów mogą być użyte na wybrane przez Fundację, dowolne działanie związane z realizacją celów Fundacji, o ile ofiarodawcy nie postanowili inaczej.
+4. Fundacja może tworzyć fundusze, w tym o charakterze celowym, zgodnie z wolą ofiarodawcy sygnowane jego nazwiskiem lub nazwą.
+5. Przyjęcie przez Fundację darowizny lub spadku nie może spowodować przejęcia przez Fundację długów przewyższających wartość mienia przyjmowanego.
+6. W sprawach przyjęcia darowizn i dziedziczenia oświadczenia wymagane przepisami prawa składa Zarząd Fundacji.
+
+§14 Działalność gospodarcza #DoDyskusji
+1. Fundacja prowadzi / nie prowadzi działalności gospodarczej. (????)
+2. Przychód z działalności odpłatnej pożytku publicznego służy wyłącznie realizacji celów Fundacji.
+
+§15 Księgi rachunkowe
+Fundacja prowadzi gospodarkę finansową i rachunkowość według zasad obowiązujących osoby prawne.
+
+§16 Powiernictwo środków
+Fundacja może gromadzić swoje fundusze w walucie polskiej oraz w walutach obcych we właściwych bankach zgodnie z przepisami prawa.
+
+§17 Odpowiedzialność za zobowiązania
+Fundacja odpowiada za swoje zobowiązania całym majątkiem.
+
+Rozdział IV: Organy Fundacji
+
+§18 Zarząd
+1. Organem Fundacji jest Zarząd Fundacji.
+2. Zarząd kieruje działaniami Fundacji i reprezentuje ją na zewnątrz.
+
+§19 Fundatorzy
+1. Fundator do wykonywania swoich uprawnień może powołać pełnomocnika.
+2. W przypadku śmierci Fundatora uprawnienia przewidziane w niniejszym statucie dla Fundatora, wykonuje jego pełnoletnie dziecko, łącznie pełnoletnie dzieci lub do czasu uzyskania pełnoletności ich prawni opiekunowie. Jeżeli Fundator nie posiadał dzieci, jego obowiązki wykonuje żyjący małżonek, albo łącznie rodzeństwo Fundatora, albo łącznie pełnoletnie dzieci rodzeństwa Fundatora lub do czasu uzyskania pełnoletności ich prawni opiekunowie.
+
+§20 Członkowie zarządu #DoDyskusji
+1. Członkowie Zarządu Fundacji powoływani i odwoływani są przez Fundatorów w drodze zgodnego oświadczenia na czas wskazany w akcie powołania.
+2. Fundator może zostać członkiem Zarządu Fundacji.
+3. Zarząd Fundacji liczy od ... do ... (???) członków.
+4. Członkowie Zarządu Fundacji mogą otrzymywać wynagrodzenie z tytułu pełnienia swoich funkcji.
+5. Członkostwo w Zarządzie Fundacji ustaje w przypadku odwołania, pisemnej rezygnacji z członkostwa lub śmierci Członka.
+
+§21 Zadania zarządu
+1. Zarząd Fundacji kieruje działalnością Fundacji i reprezentuje ją na zewnątrz.
+2. Do zadań Zarządu należy w szczególności:
+	1) Kierowanie bieżącą działalnością Fundacji;
+	2) Podejmowanie decyzji w sprawie prowadzenia działalności gospodarczej;
+	3) realizacja celów statutowych Fundacji,
```

Comment:
czy to nie jest oczywiste?

Reply:
1. W dokumentach prawnych nie ma pojęć oczywistych :P ZAMKNIĘTE
2. natomiast znalazłem kwestię, że ten punkt jest zależny od decyzji na temat działalności gospodarczej OTWARTE, DEPENDENCY Art. 14